examples/login-form/config.php

```
@@ -0,0 +1,99 @@
+    return null;
+}
+
+/**
+ * ログインしているかチェック / Check if user is logged in
+ *
+ * @return bool ログイン済みならtrue / True if logged in
+ */
+function isLoggedIn(): bool
+{
+    return isset($_SESSION['user']);
+}
```

PHPMD (2.15.0)

isLoggedIn accesses the super-global variable $_SESSION. (undefined)

(Superglobals)